.luacheckrc

```diff
@@ -0,0 +1,24 @@
+unused_args = false
+allow_defined_top = true
+max_line_length = 999
+
+ignore = {
+    "can_grow",
+    "x",
+    "y",
+    "z",
+}
+
+globals = {
+	"minetest"
```

Comment:
Mixed tabs+spaces throughout the entire file. I suggest converting your template to spaces so this isn't an issue.